### DISCUSSION TITLE
Justwatch watchlist question

### DISCUSSION THREAD

**Discussion by bagajohny** [DiegoFleitas/letterboxd-movie-justwatch#28] [Q&A] [answered]
<Explain your issue in general terms, do you have a justwatch watchlist you want to export to where? what have you tried? Of the top of my head I think a JS bookmarklet might be the easier approach but that depends on what you want and what you have>

I have a JustWatch watchlist that I want to export to Simkl. I have tried the method [of this repo.](https://github.com/ijanos/justwatchdumper) from ijanos. But I am not getting the desired output when I run the code. I get following output
![image](https://github.com/DiegoFleitas/letterboxd-movie-justwatch/assets/101859073/c1d8448b-f634-4d22-b03b-ee445c2a9469)

I hope you can help me. I do not understand programming but I can follow instructions and execute them.

**Comment by DiegoFleitas** [comment, marked as answer]
The JS you provided works for me

![imagen](https://github.com/DiegoFleitas/letterboxd-movie-justwatch/assets/15369935/36331c1b-3498-4d57-8d84-0b5bc05f0324)

But you need to execute it in the right page & have the right option selected ex: if you are not on the watchlist page or if you selected tv shows (series on my language) but have no tv shows on your watchlist then it won't output anything.

![imagen](https://github.com/DiegoFleitas/letterboxd-movie-justwatch/assets/15369935/53830bf5-f540-44bd-8ad3-5d0f658f6fe7)

**Comment by bagajohny** [comment]
I see. Now I understand the problem. I have been trying to execute it on the "Seen" page (shown as "Visto" in your language). Thanks for clearing my issue. ~~Can you please help me here? I have more titles under the "Seen" label than under the "Watchlist" label. How do I modify the code to make it work for the "Seen" label? I have marked the label in the screenshot below,~~

!
Edit: I was able to get the list of all URLs in a txt file. Thanks again!
